core/auth/src/main/java/software/amazon/awssdk/auth/signer/internal/AbstractAws4Signer.java

```diff
@@ -176,24 +176,19 @@ protected abstract void processRequestPayload(SdkHttpFullRequest.Builder mutable
     private String createCanonicalRequest(SdkHttpFullRequest.Builder request,
                                           String contentSha256,
                                           boolean doubleUrlEncode) {
-        final String canonicalRequest = request.method().toString() +
-                                        SignerConstant.LINE_SEPARATOR +
-                                        // This would optionally double url-encode the resource path
-                                        getCanonicalizedResourcePath(request.encodedPath(), doubleUrlEncode) +
-                                        SignerConstant.LINE_SEPARATOR +
-                                        getCanonicalizedQueryString(request.rawQueryParameters()) +
-                                        SignerConstant.LINE_SEPARATOR +
-                                        getCanonicalizedHeaderString(request.headers()) +
-                                        SignerConstant.LINE_SEPARATOR +
-                                        getSignedHeadersString(request.headers()) +
-                                        SignerConstant.LINE_SEPARATOR +
-                                        contentSha256;
 
-        if (LOG.isDebugEnabled()) {
-            LOG.debug("AWS4 Canonical Request: '\"" + canonicalRequest + "\"");
-        }
```

Comment:
Could we log this at the trace level, or put it behind an "insecure" `SdkStandardLogger`? This is pretty useful for debugging signature issues.

Reply:
yeah, updated it to trace level